src/ai/backend/storage/api/manager.py

```diff
@@ -776,15 +779,53 @@ class Params(TypedDict):
     ) as params:
         await log_manager_api_entry(log, "mkdir", params)
         ctx: RootContext = request.app["ctx"]
+        vfid = params["vfid"]
+        parents = params["parents"]
+        exist_ok = params["exist_ok"]
+        relpath = params["relpath"]
+        relpaths = relpath if isinstance(relpath, list) else [relpath]
+        failure_tasks: List[str] = []
+        success_tasks: List[str] = []
```

Comment:
```suggestion
        failure_tasks: list[str] = []
        success_tasks: list[str] = []
```

https://docs.python.org/3/library/typing.html#typing.List
`List` is deprecated since version 3.9